internal/security/config/command/proxy/adduser/command.go

```diff
@@ -266,17 +198,19 @@ func (c *cmd) ExecuteAddJwt() (int, error) {
 	return interfaces.StatusCodeExitNormal, nil
 }
 
+// ExecuteAddOAuth2 (placeholder)
 func (c *cmd) ExecuteAddOAuth2() (statusCode int, err error) {
-	if err := c.createConsumer(); err != nil {
-		return interfaces.StatusCodeExitWithError, err
-	}
-	if err := c.addUserToGroup(); err != nil {
-		return interfaces.StatusCodeExitWithError, err
-	}
 
 	// Create an OAuth application
 	// https://docs.konghq.com/hub/kong-inc/oauth2/#create-an-application
 
+	// Read in JWT
+	jwtBytes, err := ioutil.ReadFile(c.jwtFile)
+	if err != nil {
+		return interfaces.StatusCodeExitWithError, fmt.Errorf("failed to read JWT file: %w", err)
```

Comment:
Could you put some comments to explain why we don't `createConsumer` and `addUserToGroup` in this `ExecuteAddOAuth2` function?

Reply:
So...I was going to ask y'all about this. I feel like these processes should just be reworked since we're in the mix of it right now - unless we absolutely want to close this out ASAP. 

It is no longer necessary (after my updates) to create a consumer or group (unless someone wanted to create specific users?). The consumer and group are created in the declarative config file I generate in secret-store-setup - thus no longer requiring these steps.

So now, executing an "adduser" command seems misleading, since it's really only associating a JWT or OAuth token to the "gateway" user that's already been pre-configured. I really don't see the point in having a user other than "admin" and "gateway" for our purposes. Thoughts?

@lenny-intel @bnevis-i

Reply:
I agree.  Let's take this out the extra cruft and simplify while we can.  The kong design allows multiple credentials to be associated with a single consumer, so we might as well run with it.  This won't make any meaningful difference (other than that it would look like the same user from a logging perspective) to the functionality of the gateway.

Reply:
@beaufrusetta More tricky, however, will be the "delete user" functionality.   Adding/deleting consumers takes all the auth methods with them.  But adding/deleting auth methods by themselves might be more tricky.

Reply:
So - I believe this has been resolved when I went back in and restored that functionality. This way, the user is allowed to create new users/groups with different tokens if they want to - not sure why they would, because at this point they have no ACL to control...but...I'm just following the direction from the majority here... ;)